File: iPad/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/comm/QuoteSizeData.m
//
//  QuoteSizeData.m
//  VerBank-CSTSv3-IOSClientApi
//
//  Created By Zhanglei on 15/6/30.
//  Copyright (c) 2015年 allone. All rights reserved.
//

#import "QuoteSizeData.h"
static NSString * jsonId = @"QuoteSizeData";

static NSString * instrument = @"1";
static NSString * priceType = @"2";
static NSString * ammount = @"3";
@implementation QuoteSizeData
- (id)init{
    self = [super init];
    if(self != nil)
    {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}
@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/CSTS3/comm/struct/QuoteSizeData.java
package jedi.ex01.CSTS3.comm.struct;


public class QuoteSizeData extends allone.json.AbstractJsonData {

	public static final String jsonId = "19";

	public static final String instrument = "1";
	public static final String priceType = "2";
	public static final String ammount = "3";

	public QuoteSizeData(){
		super();
		setEntry("jsonId", jsonId);
	}

	public String getInstrument() {
		try {
			String data=getEntryString(QuoteSizeData.instrument);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setInstrument(String instrument) {
		setEntry(QuoteSizeData.instrument, instrument);
	}

	public int getPriceType() {
		try {
			int data=getEntryInt(QuoteSizeData.priceType);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setPriceType(int priceType) {
		setEntry(QuoteSizeData.priceType, priceType);
	}

	public double getAmmount() {
		try {
			double data=getEntryDouble(QuoteSizeData.ammount);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setAmmount(double ammount) {
		setEntry(QuoteSizeData.ammount, ammount);
	}


}